tests/upgrade/test_crossgrade.sh

```diff
@@ -328,6 +329,9 @@ echo "Waiting for traffic to settle..."
 sleep 20
 
 installIstioSystemAtVersionHelmTemplate "${TO_HUB}" "${TO_TAG}" "${TO_PATH}"
+# Wait a bit for the pod status to show up.
```

Comment:
Is there any reason we're checking individual pod status vs. `kubectl rollout status deployment <name>`? I think the latter may handle this case.